tasks/install.yml

```diff
@@ -36,3 +36,11 @@
     name: "{{ mattermost_user }}"
     system: yes
     createhome: no 
+
+- name: Create version file
```

Comment:
Please also check the contents of the file

Reply:
Because of `force: yes`, the `version` file contents are overwritten every time. (I did also test this manually.)

Reply:
I think he means that you should check the contents when installing or updating mattermost.

Reply:
I added a corresponding commit and updated the PR description.